tmc-langs-rust/src/main/java/fi/helsinki/cs/tmc/langs/rust/RustPointsParser.java

```diff
@@ -0,0 +1,59 @@
+package fi.helsinki.cs.tmc.langs.rust;
+
+import fi.helsinki.cs.tmc.langs.domain.ExerciseDesc;
+import fi.helsinki.cs.tmc.langs.domain.TestDesc;
+
+import com.google.common.base.Optional;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableList.Builder;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Scanner;
+
+public class RustPointsParser {
+
+    /**
+     * Parses a file containing information of points of exercise.
+     */
+    public Optional<ExerciseDesc> parse(Scanner scanner, String exerciseName) {
+        Map<String, List<String>> map = new HashMap<>();
+        while (scanner.hasNextLine()) {
+            String[] keyValue = scanner.nextLine().split(" = ");
+            if (keyValue.length == 1) {
```

Comment:
Perhaps silly, but is it possible for the input to be malformed in a way that we'd get a 3-way (or larger) split, and would that cause issues further down the road? Essentially, is there any reason to not just have `if (keyValue.length != 2)` here?